types/node-pushnotifications/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/node-pushnotifications",
-    "version": "1.0.9999",
+    "version": "3.0.9999",
```

Comment:
Shouldn't this match the current version of the package?  I see `3.1.1` is the latest